sympy/physics/control/tests/test_lti.py

```diff
@@ -1679,3 +1679,35 @@ def test_StateSpace_functions():
     assert ss3.input_matrix == Matrix([[0, 0], [1, 0], [0, 1], [0, 0]])
     assert ss3.output_matrix == Matrix([[0, 1, 0, 0], [0, 0, 1, 0]])
     assert ss3.feedforward_matrix == Matrix([[0, 0], [0, 1]])
+
+
+def test_Feedback_as_TransferFunction():
```

Comment:
The changes you made to operations of Feedback like `add`, `sub`, `mul` and `div` don't reflect here. Can you cover those in your unit tests ?

Reply:
Yes , Thanks for mentioning I've added tests that cover these test cases also.